src/Laravel/Controller/ApiPlatformController.php

```diff
@@ -64,11 +64,11 @@ public function __invoke(Request $request): Response
             $operation = $operation->withValidate(!$request->isMethodSafe() && !$request->isMethod('DELETE'));
         }
 
-        if (null === $operation->canRead() && $operation instanceof HttpOperation) {
+        if ($operation instanceof HttpOperation && null === $operation->canRead()) {
```

Comment:
maybe we should do this just once at the top of the __invoke function, it's not possible that the operation isn't an instance of HttpOperation.